earth_enterprise/BUILD.md

```diff
@@ -31,7 +31,7 @@ platforms on how to setup the dependencies, tools, and compilers.
         more error prone):
 
         ```bash
-        git clone git@github.com:google/earthenterprise.git
+        git clone https://github.com/google/earthenterprise.git
```

Comment:
We should change the git command a few lines down to use https as well to be consistent.